testsuite/classlibrary/TestConversions.sc

```diff
@@ -0,0 +1,62 @@
+TestConversions : UnitTest {
```

Comment:
"Conversions" could cover a lot of methods, not all of which should be grouped together IMO. Can we rename this `TestCollectionConversions`? I think Collections are a good functionally contained unit to test on.